packages/plugin-link/src/LinkXmlDomParser.ts

```diff
@@ -19,6 +20,11 @@ export class LinkXmlDomParser extends FormatXmlDomParser {
         }
         const children = await this.engine.parse(...item.childNodes);
         this.applyFormat(link, children);
+        for (const node of children) {
```

Comment:
Add a comment:
`// The applyFormat helper only applies the format to InlineNodes.`